homeassistant/components/withings/sensor.py

```diff
@@ -259,6 +259,13 @@ class WithingsMeasurementSensorEntityDescription(SensorEntityDescription):
         native_unit_of_measurement=PERCENTAGE,
         entity_registry_enabled_default=False,
     ),
+    MeasurementType.BASAL_METABOLIC_RATE: WithingsMeasurementSensorEntityDescription(
```

Comment:
Is it useful to set a state_class?

Reply:
SensorDeviceClass.ENERGY - BMR are energy units per unit time, in this case Withings is showing kcal / day if that is supported. The sensor device class represents power over time which sounds about right and is the only class offering calories as a unit. I would say your state class should be MEASURMENT as it does not represent a total amount and using MEASUREMENT + ENERGY will allow opt-in for statistics which many people like.